pallets/pool-system/src/mock.rs

```diff
@@ -290,6 +337,36 @@ impl<T> PreConditions<T> for Always {
 	}
 }
 
+impl pallet_mock_change_guard::Config for Runtime {
+	type Change = pallet_pool_fees::types::Change<Runtime>;
+	type ChangeId = H256;
+	type PoolId = PoolId;
+}
+
+parameter_types! {
+	pub const MaxPoolFeesPerBucket: u32 = cfg_primitives::constants::MAX_POOL_FEES_PER_BUCKET;
+	pub const PoolFeesPalletId: PalletId = cfg_types::ids::POOL_FEES_PALLET_ID;
+}
+
+impl pallet_pool_fees::Config for Runtime {
```

Comment:
From the `system-pool` perspective I think you only need to check that the calls to `on_closing()`, `on_execution_pre_fulfillments()` and `add_fees()` were done, right? I think a pallet mock of both traits could simplify a lot the `pool-system` tests

Reply:
Ok 👍🏻